application/controllers/ModelController.php

```diff
@@ -511,6 +511,10 @@ public function createAction()
                 );
                 $this->view->errorFlag = true;
                 return;
+            } else if (!Erfurt_Uri::check($newModelUri)) {
+                $this->_owApp->appendMessage(
+                    new OntoWiki_Message('The given String is no legal URI (http://...)', OntoWiki_Message::ERROR)
```

Comment:
The hint "http://…" connotes, that the algorithm already identified the missing protocol part, while there might be other cases for the value not being a valid URI or IRI.